lib/atm.rb

```diff
@@ -11,11 +11,12 @@ def initialize
 
    def withdraw(amount, pin, expiry, account)
       case
-      when negative_value(amount) then return_error_message(:negative_amount)
-      when !check_pin(pin, account.pin) then return_error_message(:wrong_pin)
+      when negative_value?(amount) then return_error_message(:negative_amount)
+      when incorrect_pin?(pin, account.pin) then return_error_message(:wrong_pin)
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)